<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myProgressBox</class>
 <widget class="QDialog" name="myProgressBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#myProgressBox{
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
	background-color: rgb(55, 63, 81);
}</string>
   </property>
   <widget class="QLabel" name="lab_title">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>8</y>
      <width>76</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Agency FB&quot;;
background-color: rgb(55, 63, 81);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>正在读取</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolTip{border:1px solid rgb(118, 118, 118); background-color: #ffffff; color:#484848; font-size:12px;}
QPushButton:hover{border: no;
background-image:url(:/images/btnClose_mini.png);}
QPushButton{border: no;
background-image: url(:/images/btnClose_mini_norm.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>275</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{ font: bold large 'SourceHanSansK-Regular';font-size:14px;
                                border: 0px solid;
                                background-color:rgba(255, 30, 30, 1);
                                color:rgba(255, 255, 255,1);
                                border-radius: 2px;
                                }
                                QPushButton:disabled{
                                background-color:#DCDCDC;
                                border-radius:2px;
                                color:#FFFFFF;    
                                }
                                QPushButton:hover{
                                border: 0px solid;
                                background-color:rgba(255, 87, 87,1);
                                color:rgba(255, 255, 255,1);
                                border-radius:2px;
                                opacity:0.2;
                                }
                                QPushButton:pressed{
                                background-color:rgba(56, 60, 73,1);
                                color:rgba(255, 255, 255,1);
                                border-radius:2px;
                                }</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_num">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>79</y>
     <width>151</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>48</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgba(88,164,176,1);
font: 48pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="picProgress" name="widget_MyProgress" native="true">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>150</y>
     <width>482</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>482</width>
     <height>17</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>482</width>
     <height>17</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lab_info">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>193</y>
     <width>438</width>
     <height>34</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>34</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>34</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 13pt &quot;等线&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>设备数据读取中请等待..(请勿在读取过程中断开连接!)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_read">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>175</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 12pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string> 共需读取:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_readCount">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>175</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(88,164,176,1);
font: 12pt &quot;等线&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>12345</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_read_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>175</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;等线&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>条</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_now">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>175</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;等线&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>当前读取数据编号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_nowCount">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>175</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(88,164,176,1);
font: 12pt &quot;等线&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>*****</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labInfo">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>329</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>picProgress</class>
   <extends>QWidget</extends>
   <header>myProgressBox/picprogress.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
